lib/keypair/spec.ex

```diff
@@ -1,21 +1,26 @@
 defmodule Stellar.KeyPair.Spec do
   @moduledoc """
   Specifies the behaviour for KeyPair generators.
-  This Library allows you to use any crypto package of your choice. The default is Ed25519.
+  This Library allows you to use any crypto package of your choice.
+  The default is Ed25519.
```

Comment:
Given there are defined callbacks specifically for Ed25519, it seems like there is no natural choice to use any other crypto package.

Reply:
@luishurtado you're totally right. Change is scheduled in #80